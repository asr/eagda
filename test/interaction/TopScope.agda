-- Check that F is in not in scope at the top-level (but in the hole).
module _ where

private
  postulate F : Set â†’ Set

Hole : Set
Hole = {!!}
