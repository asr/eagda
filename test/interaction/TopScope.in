top_command (cmd_load currentFile [])
goal_command 0 (cmd_infer Normalised) "F"
top_command (cmd_infer_toplevel Normalised "F")
