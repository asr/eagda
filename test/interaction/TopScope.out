Agda2> (agda2-status-action "")
(agda2-info-action "*Type-checking*" "" nil)
(agda2-highlight-clear)
(agda2-info-action "*Type-checking*" "Checking TopScope (TopScope.agda). " t)
(agda2-info-action "*Type-checking*" "Finished TopScope. " t)
(agda2-status-action "")
(agda2-info-action "*All Goals*" "?0 : Set " nil)
((last . 1) . (agda2-goals-action '(0)))
Agda2> (agda2-status-action "")
(agda2-info-action "*Inferred Type*" "Set â†’ Set" nil)
((last . 1) . (agda2-goals-action '(0)))
Agda2> (agda2-info-action "*Error*" "1,1-2 Not in scope: F at 1,1-2 when scope checking F" nil)
(agda2-highlight-load-and-delete-action)
(agda2-status-action "")
Agda2> 